core/src/test/scala/unit/kafka/message/MessageTest.scala

```diff
@@ -107,32 +111,31 @@ class MessageTest extends JUnitSuite {
   }
 
   @Test(expected = classOf[IllegalArgumentException])
-  def testInvalidTimestampAndMagicValueCombination() {
-      new Message("hello".getBytes, 0L, Message.MagicValue_V0)
+  def testInvalidTimestampAndMagicValueCombination() = {
+      val msg = new Message("hello".getBytes, 0L, Message.MagicValue_V0)
```

Comment:
When I added `=`, the return type of the function changed from `Unit` to `Message` (see comment above). Now I see it would be much easier to write `:Unit = { ... }` to achieve the same.